repos/base-hw/src/bootstrap/spec/arm/cortex_a9_mmu.cc

```diff
@@ -109,6 +109,7 @@ unsigned Bootstrap::Platform::enable_mmu()
 
 	Cpu::Sctlr::init();
 	Cpu::Cpsr::init();
+	Actlr::disable_smp();
```

Comment:
Hi @ValiValpas I wonder why do you need to first unset the SMP bit when it is not initialized yet. Can you give me some insights?

Reply:
More are less, I figured this out by trial and error. I noticed that the SMP bit of the primary CPU was enabled when coming from uboot and that it got stuck at waiting for one of the Cpu_counter variables. In contrast, it worked fine when booting with the Lauterbach and all SMP bits disabled.

I guess, one could read this from Section 5.3.4 of the Cortex-A9 mpcore Technical Reference Manual ("Multiprocessor bring-up") as it states the SMP mode shall be enabled. This supposedly implies that it shall be disabled before ;-)